src/pyspextool/telluric/make_telluric_spectra.py

```diff
@@ -110,38 +115,92 @@ def make_telluric_spectra(intensity_unit:str='W m-2 um-1',
             #
             # Change units to those requested by the user
             #
-
-            flux = result[0]
-            unc = result[1]
-            vega = result[2]
             
             flux = convert_fluxdensity(standard_wavelength,
-                                       result[0],'um','erg s-1 cm-2 A-1',
+                                       result[0],
+                                       'um','erg s-1 cm-2 A-1',
                                        intensity_unit)
             
             unc = convert_fluxdensity(standard_wavelength,
-                                      result[1],'um','erg s-1 cm-2 A-1',
+                                      result[1],
+                                      'um','erg s-1 cm-2 A-1',
                                       intensity_unit)
             
             vega = convert_fluxdensity(standard_wavelength,
-                                       result[2],'um','erg s-1 cm-2 A-1',
+                                       result[2],
+                                       'um','erg s-1 cm-2 A-1',
                                        intensity_unit)
             
             # Updates labels
             
             telluric_spectra[i,1,:] = flux
             telluric_spectra[i,2,:] = unc
-            vega_spectra[i,1,:] = vega
+            model_spectra[i,1,:] = vega
             
-        #
-        # Store the Vega results
-        #
+    if tc.state['type'] == 'basic':
 
-        tc.state['vega_spectra'] = vega_spectra    
+        tc.state['intensity_unit'] = intensity_unit
+        
+        for i in range(tc.state['object_norders']):    
 
-    else:
+            standard_wavelength = tc.state['standard_spectra'][i,0,:]
+            standard_fluxdensity = tc.state['standard_spectra'][i,1,:]
+            standard_uncertainty = tc.state['standard_spectra'][i,2,:]
+
+        
+            telluric_spectrum = 1/standard_fluxdensity
+            telluric_unc = 1/standard_fluxdensity**2 * standard_uncertainty
+
+            # Flux calibrate using a Planck function
+
+            l5556 = 5556.e-4
+            bb_5556 = 1.1910e8 / l5556**5 / \
+            (np.exp(14387.7/l5556/tc.state['standard_teff'])-1)
+
+            scale =  3.46e-9*10**(-0.4*(tc.state['standard_vmag']-0.03)) / \
+                bb_5556
+            
+            wave = telluric_spectra[i,0,:]
+            planck = 1.1910e8 / wave**5 / \
+                (np.exp(14387.7/wave/tc.state['standard_teff'])-1)*scale
```

Comment:
Use astropy Blackbody: https://docs.astropy.org/en/stable/api/astropy.modeling.physical_models.BlackBody.html

Reply:
I have tried to avoid using other packages as much as possible. Things change, and this way if I write it, then I know what it does. Obviously that doesn't work for everything, but for this kind of stuff I want to do it this way.

Reply:
This function has way too many numbers hard coded and is extremely fragile.  I'm also gonna hold my ground on this one.

Reply:
it is a simplified version of the Planck function that gives you W m-2 um-1 directly from wavelengths in units of microns, which is what we have.  The astropy version returns a per frequency distribution and we need a per wavelength.  So either way there is a bunch of unit conversions that have to be done.  

I should note that I do use astropy for conversions in my units.convert_fluxdensity so I am not completely against it.  But to me (and I need to clean up the comments there), this is actually easier for someone to read because everything is there.